sormas-api/src/main/resources/strings.properties

```
@@ -969,6 +970,7 @@ messageUserSyncPartiallySuccessful = <b>Sync partially successful!</b><br/>The s
 messageUserSyncSuccessful = <b>Sync successful!</b><br/>All users have been synced. You can now close this window.
 messageUserRightsExportFailed = Please contact an admin and notify them about this problem
 messageNoLabMessagesSelected = You have not selected any lab messages
+messageLabMessageDownloadFailed = Something went wrong when trying to download the lab message
```

Could you maybe change this to:
```suggestion
messageExternalLabResultsAdapterNotFound = The external lab results adapter could not be found. This probably means you system is not configured correctly. Please contact you system administrator.
```
This would be a little more informative and a good foundation for #5338.